pkgs/tools/security/age/default.nix

```diff
@@ -2,9 +2,9 @@
 
 buildGoModule rec {
   pname = "age";
-  version = "1.0.0-beta2";
+  version = "2020-03-25";
```

Comment:
Let's include a `master-` or similar prefix here to make it clearer what version this is, otherwise LGTM.